frontend/server/src/Controllers/User.php

```diff
@@ -3977,44 +3999,75 @@ public static function getRankForTypeScript(\OmegaUp\Request $r) {
             $params[ 'filter'] = $currentFilter;
         }
 
-        try {
-            $r->ensureIdentity();
-        } catch (\OmegaUp\Exceptions\UnauthorizedException $e) {
-            // Do nothing. Not logged user can access here
-            $r->identity = null;
-        }
-        if (is_null($r->identity)) {
-            [
-                'ranking' => $ranking,
-                'pager' => $pager,
-            ] = self::getRankByProblemsSolvedWithPager(
-                loggedIdentity: null,
-                filteredBy: $currentFilter,
-                offset: $page,
-                rowCount: $length,
-                params: $params,
-            );
+        [
+            'ranking' => $ranking,
+            'pager' => $pager,
+        ] = self::getRankByProblemsSolvedWithPager(
+            loggedIdentity: null,
+            filteredBy: $currentFilter,
+            offset: $page,
+            rowCount: $length,
+            params: $params,
+        );
 
-            return [
-                'templateProperties' => [
-                    'payload' => [
-                        'page' => $page,
-                        'length' => $length,
-                        'filter' => $currentFilter,
-                        'availableFilters' => $availableFilters,
-                        'isIndex' => false,
-                        'isLogged' => false,
-                        'ranking' => $ranking,
-                        'pagerItems' => $pager,
-                        'lastUpdated' => $ranking['rank'][0]['timestamp'] ?? null,
-                    ],
-                    'title' => new \OmegaUp\TranslationString(
-                        'omegaupTitleUsersRank'
-                    )
+        return [
+            'templateProperties' => [
+                'payload' => [
+                    'page' => $page,
+                    'length' => $length,
+                    'filter' => $currentFilter,
+                    'availableFilters' => $availableFilters,
+                    'isIndex' => false,
+                    'isLogged' => false,
+                    'ranking' => $ranking,
+                    'pagerItems' => $pager,
+                    'lastUpdated' => $ranking['rank'][0]['timestamp'] ?? null,
                 ],
-                'entrypoint' => 'users_rank',
-            ];
+                'title' => new \OmegaUp\TranslationString(
+                    'omegaupTitleUsersRank'
+                )
+            ],
+            'entrypoint' => 'users_rank',
+        ];
+    }
+
+    /**
+     * When user is logged in, prepare all the properties to be sent to the rank table view via TypeScript
```

Comment:
En PHP  no tenemos un linter que valide la longitud de las líneas en los comentarios, entonces como sugerencia para que se vea mejor ordenado el texto lo puedes separar en varias líneas de la siguiente forma:
```suggestion
     * When user is logged in, prepare all the properties to be sent to the
     * rank table view via TypeScript
```